CHANGELOG.md

```diff
@@ -366,7 +435,8 @@ See the comparison for full list.
 
 Initial tagged release.
 
-[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.11.0...HEAD
+[Unreleased]: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/compare/0.12.0...HEAD
```

Comment:
Side note, but couldn't this be changed to `master...HEAD`, and then it wouldn't have to be updated every time? Or is the goal to have it reflect whatever version of the changelog is being viewed?